VKProject/Src/Pages/MyPage.cs

```diff
@@ -22,6 +30,14 @@ public static string GetCreatorNameFromPost(string postId)
 
     public static string GetTextFromPost(string postId)
     {
+        try
+        {
+            BrowsersService.Driver.SwitchTo().Alert().Accept();
+        }
+        catch (NoAlertPresentException)
+        {
```

Comment:
what's happening when you are trying to handle the exception? Maybe there's better throw any?